contents/blog/early-stage-analytics.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Early-stage startups need analytics, here’s why (and what matters most)"
+date: 2023-02-07
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-blog-image.png
+featuredImageType: full
+category: Startups
+tags:
+ - Guides
+---
+
+Starting a startup is trying to find the exit of a dark cave. You attempt a lot of things, but it is mostly dark. Sometimes you hear sounds leading you somewhere, but they can also be misleading. In this cave, analytics is like a flashlight. It doesn’t automatically show you the exit, but it does provide clarity on where you are going. With the light of analytics, you stumble less and spend less time moving in the wrong direction.
+
+![Flashlight](../images/blog/early-stage-analytics/light.png)
+
+Our team works with a lot of high-growth startups (they are [our ideal customers](/handbook/strategy/ideal-customer-persona)). We succeed when they succeed. We believe analytics is a key part of finding early success if done right. This post goes over why we think your startup needs analytics, and what exactly those analytics should look like at an early stage.
+
+## Why do analytics matter for startups?
```

Comment:
This section is ok, but it would be with some more concrete examples either from our own experience, or from those of our users.

Reply:
Added one on OKRs